CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,4166 @@
+	option = {
+		name = "Democratic Centre (DC-PLI)"
+		ruling_party_ideology = conservative
+		
+		any_pop = {
+			ideology = { factor = 0.04 value = conservative }
+			ideology = { factor = 0.04 value = big_tent }
+		}
+		
+		ai_chance = { factor = 0.1 }
+	}
+	
+}
+
+#Elections 5
+country_event = {
+ 	id = 8020418
+  	title = "$YEAR$ Italian General Election"
+ 	desc = EVT_8020418_DESC
+  	picture = "ITA_1972_elections"
+
+	trigger = {
+		tag = ITA
+		NOT = { has_global_flag = 1992_start_date_flag }
+		
+		government = democracy
+		is_vassal = no
+		war = no
+		
+		OR = { ruling_party_ideology = liberal ruling_party_ideology = socialist ruling_party_ideology = conservative }
+		
+		has_country_flag = ITA_Democracy_Restored
+		has_country_flag = ITA_Democracy_Elections
+		has_country_flag = ITA_Democratic_Opposition
+		has_global_flag = ITA_Operation_Tora_Tora
+	}
+	
+	mean_time_to_happen = { months = 60 }
+	
+	immediate = { clr_country_flag = ITA_Democracy_Elections set_country_flag = ITA_Democracy_Elections2 }
+	
+	option = {
+		name = "New Centre (PRI-PSDI)"
+		ruling_party_ideology = liberal
+		
+		any_pop = {
+			ideology = { factor = 0.04 value = big_tent }
+			ideology = { factor = 0.07 value = liberal }
+		}
+		
+		ai_chance = { factor = 0.6 }
+	}
+	
+	option = {
+		name = "Left Cartel (PSI-PR)"
+		ruling_party_ideology = socialist
+		
+		any_pop = {
+			ideology = { factor = 0.04 value = progressive }
+			ideology = { factor = 0.06 value = socialist }
+			ideology = { factor = 0.01 value = communist }
+		}
+		
+		ai_chance = { factor = 0.3 }
+	}
+	
+	option = {
+		name = "Democratic Centre (DC-PLI)"
+		ruling_party_ideology = conservative
+		
+		any_pop = {
+			ideology = { factor = 0.03 value = conservative }
+			ideology = { factor = 0.04 value = big_tent }
+		}
+		
+		ai_chance = { factor = 0.1 }
+	}
+	
+}
+
+#Democratic Reorganization - Economy
+country_event = {
+  	id = 8020419
+  	title = "Socio-Economic Reorganization of the Country"
+  	desc = EVT_8020419_DESC
+  	picture = "gelli_andreotti"
+  
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1973
+		NOT = { year = 1992 }
+		
+		government = populist_dictatorship 
+		is_vassal = no
+		war = no
+		
+		ruling_party_ideology = conservative
+		
+		has_country_modifier = dirty_war
+		
+		has_country_flag = ITA_P2
+		has_global_flag = ITA_Operation_Tora_Tora
+	}
+
+	mean_time_to_happen = { months = 6 }
+	
+	immediate = { set_country_flag = ITA_P2_2 }
+	
+	option = {
+		name = "Enact the reforms"
+		prestige = 25
+		
+		money = 100000
+		
+		any_pop = {  
+		    limit = { strata = poor }
+			consciousness = 4
+			militancy = 2
+		}
+		
+		any_country = {
+			limit = { has_country_modifier = alliance_nato } 
+			relation = { who = THIS value = 25 } 
+		}
+		
+		add_country_modifier = { name = economic_boom duration = 365 }
+		add_country_modifier = { name = bilateral_trade_pact_gp duration = 365 }
+		add_country_modifier = { name = local_stock_exchange duration = 365 }
```

Comment:
local_stock_exchange is only for the province scope so this has to be deleted to edited